constants.go

```diff
@@ -32,6 +32,7 @@ const (
 	MIMETextHTMLCharsetUTF8        = "text/html; charset=utf-8"
 	MIMETextPlainCharsetUTF8       = "text/plain; charset=utf-8"
 	MIMETextJavaScriptCharsetUTF8  = "text/javascript; charset=utf-8"
+	MIMETextCSSCharsetUTF8         = "text/css; charset=utf-8"
```

Comment:
pls also add it without charset UTF-8

Reply:
Sorry but I wanna understand it. Why should we remove it?

Reply:
not to be removed, probably a misunderstanding

i would like that in addition to the already added new mimetype, which was explicitly added with the encoding, the same is added without the encoding

`MIMETextCSS         = "text/css"`

Reply:
Oh, I got it, it does make sense.